testament/important_packages.nim

```diff
@@ -19,7 +19,8 @@ pkg1 "BipBuffer"
 # pkg1 "blscurve" # pending https://github.com/status-im/nim-blscurve/issues/39
 pkg1 "bncurve"
 pkg1 "brainfuck", "nim c -d:release -r tests/compile.nim"
-pkg1 "bump", "nim c --gc:arc -r tests/tbump.nim", "https://github.com/disruptek/bump"
+# pending https://github.com/nim-lang/Nim/issues/16447
+#pkg1 "bump", "nimble test", "https://github.com/disruptek/bump"
```

Comment:
`"nimble test"` is the default, this should be:
```nim
# pending https://github.com/nim-lang/Nim/issues/16447
# pkg1 "bump"
```

ditto elsewhere

Reply:
You can change it if you want.  #16509 is either merged or not, but irrelevant in any event.